modules/ROOT/pages/_partials/generic-binding.adoc

```diff
@@ -4,7 +4,7 @@ include::{partialsdir}/attributes.adoc[]
 To use mobile services, you must represent your mobile app in OpenShift using a {mobile-client} and that {mobile-client} must be associated with the mobile service. 
 After binding a {mobile-client} to the {service-name} service, your mobile app can use that service immediately. 
```

Comment:
The sentence above is a bit redundant so I think we could say here "This association is called a binding and once it's done, your mobile app can use that service immediately". I think it's necessary to say what a binding is.

Reply:
Yes, good point. We don't really make the leap from the association to the binding.